pkg/cmd/pulumi/org_search.go

```diff
@@ -93,7 +125,14 @@ func (cmd *searchCmd) Run(ctx context.Context, args []string) error {
 	if err != nil {
 		return err
 	}
-	err = cmd.RenderTable(res.Resources)
+	switch cmd.outputFormat {
+	case outputFormatJSON:
+		err = cmd.RenderJSON(res.Resources)
```

Comment:
Feels like "Render" ought to be a method on "outputFormat" itself.

Reply:
That's nice! I like that.